src/__tests__/Calculate.test.js

```diff
@@ -0,0 +1,63 @@
+import calculate from '../logic/calculate';
+
+describe('Calculate Function', () => {
```

Comment:
Good job so far but we need to test all the features in `calculator.js`

We are missing test case for the following in this test file:
1. `case '%':`
2. `case '.':`